ecommerce/frontend/src/mocks/handlers.js

```diff
@@ -27,6 +29,17 @@ const url = u => config.apiUrl + u
 // comment the handler when done with it, DO NOT REMOVE IT
 
 export const handlers = [
+    rest.get(url('/vendor/:id/details'), (req, res, ctx) => {
+        const { params } = req
+        const { id } = params
+        const minId = min(id, vendorDetails.length - 1) // If request vendorId=55 then get 4 (last element)
+        console.log('returns: ', minId, vendorDetails[minId])
+        return res(
+            ctx.json({
+                data: vendorDetails[minId],
+            }),
```

Comment:
Return vendorDetails[minId] directly in the response like the backend doe

Reply:
I did.